.gitignore

```diff
@@ -40,3 +40,7 @@ wheelhouse
 james.ini
 celeryev.pid
 media/js/libs/ckeditor/source/ckbuilder
+lib/front-end
+
+# Only needed until Gulp manages all front-end build tasks
+media/js/libs/jquery.js
```

Comment:
Exactly. Depending on how our Gulp infrastructure evolves, it might even turn into something like `media/js/mdn.js.min` or similar.